platform-coop-toolkit/pwa-experiments
# Save language settings to local storage (resolve #2)

## Description

This PR uses the [Web Storage API](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Client-side_storage#Storing_simple_data_%E2%80%94_web_storage) to store a simple user setting (site language) in a key-value pair.

## Steps to test

1. Clone this branch, then spin up the site: `npm i && npm run serve`.
2. Visit the site at http://localhost:8080.
3. Select a language from the dropdown, then click save.
4. Reload the page.

**Expected behavior:** Your saved language should be selected.

## Additional information

Not applicable.

## Related issues

Not applicable.
